src/plugins/analysis/elf_analysis/code/elf_analysis.py

```diff
@@ -142,3 +146,22 @@ def _convert_address_values_to_hex(elf_dict):
             for entry in elf_dict[category]:
                 for key in {'virtual_address', 'offset'}.intersection(entry):
                     entry[key] = hex(entry[key])
+
+    def filter_modinfo(self, binary) -> Optional[str]:
```

Comment:
The type hint doesn't fit the changed code:
```suggestion
    def filter_modinfo(self, binary) -> Optional[List[str]]:
```